e2e/cli/up_test.go

```diff
@@ -37,10 +34,6 @@ var _ = Describe("up command", Ordered, func() {
 		envdApp := app.New()
 		err := envdApp.Run(append(baseArgs, "bootstrap"))
 		Expect(err).NotTo(HaveOccurred())
-		cli, err := docker.NewClient(context.TODO())
```

Comment:
We need to keep the test logic here, or the test case will fail

Reply:
How can I replicate Destroy method in tests? I saw `dockerEngine` is not exported @gaocegege

Reply:
You can define Destroy method in env.Engine. Then fake a context

```go
	c := types.Context{Runner: types.RunnerTypeDocker}
	opt := envd.Options{
		Context: c,
	}
	envdEngine, err := envd.New(clicontext.Context, opt)
	if err != nil {
		return errors.Wrap(err, "failed to create envd engine")
	}
```